test/kubernetes_test.go

```diff
@@ -228,22 +285,15 @@ var dnsTestCases = []test.Case{
 		Qname: "dns-version.cluster.local.", Qtype: dns.TypeTXT,
 		Rcode: dns.RcodeSuccess,
 		Answer: []dns.RR{
-			test.TXT("dns-version.cluster.local. 28800 IN TXT \"1.0.0\""),
-		},
-	},
-	{
-		Qname: "cluster.local.", Qtype: dns.TypeNS,
-		Rcode: dns.RcodeSuccess,
-		Answer: []dns.RR{
-			test.NS("cluster.local.          0       IN      NS      kubernetes.default.svc.cluster.local."),
+			test.TXT("dns-version.cluster.local. 28800 IN TXT \"1.0.1\""),
```

Comment:
If you use "`" (backtacks) you are use raw strings in Go, no need to escape the " in the TXT rdata.